pkg/store/storepb/custom.go

```diff
@@ -517,3 +517,34 @@ func (c *SeriesStatsCounter) Count(series *Series) {
 		}
 	}
 }
+
+func (m *SeriesRequest) ToPromQL() string {
+	grouping := strings.Join(m.QueryHints.Grouping, ",")
+	matchers := MatchersToString(m.Matchers...)
+	return fmt.Sprintf("%s %s (%s) (%s)", m.QueryHints.Func, m.QueryHints.ByOrWithout(), grouping, matchers)
+}
+
+func (m *QueryHints) IsFuncDistributive() bool {
```

Comment:
Can we make sure to add comment what do you mean distributive as it might be non trivial